crates/bevy_render/src/camera/projection.rs

```diff
@@ -70,6 +70,9 @@ pub struct CameraUpdateSystem;
 /// to recompute the camera projection matrix of the [`Camera`] component attached to
 /// the same entity as the component implementing this trait.
 ///
+/// Use the plugins [`CameraProjectionPlugin`] and `bevy::pbr::PbrProjectionPlugin` to setup the
+/// systems for your `CameraProjection` implementation.
```

Comment:
```suggestion
/// systems for your [`CameraProjection`] implementation.
```

CI will complain otherwise I think.

Reply:
Ok! Fixed : )